invoicer_project/clients/urls.py

```diff
@@ -2,14 +2,17 @@
 from .views import ClientViewSet
 from django.views.generic import TemplateView
 
-clients_methods = ClientViewSet.as_view({'post': 'create', 'get': 'list'})
 clients_list = ClientViewSet.as_view({"get": "list"})
-delete_client = ClientViewSet.as_view({'delete': 'destroy'})
+client_crud = ClientViewSet.as_view(
+    {'post': 'create', 'patch': 'partial_update', 'get': 'retrieve', 'delete': 'destroy'})
 
 urlpatterns = [
     path("home/", TemplateView.as_view(template_name="clients/clients_list.html"), name='home'),
     path("list/", clients_list, name='clients'),
-    path("client/<int:client_id>", delete_client, name='delete_client'),
-    path("client/", clients_methods, name='clients_add'), 
-    path("add/", TemplateView.as_view(template_name="clients/client_add.html"), name='render_add')
-]
\ No newline at end of file
+    path("client/<int:client_id>", client_crud, name='delete_client'),
+    path("client_edit/<int:client_id>", client_crud, name='edit_client'),
```

Comment:
do you really need different endpoint? I mean that you have endpoint `/client`.  Why doesn't reuse it?

Reply:
Done